MmSupervisorPkg/Core/Relocate/Relocate.c

```diff
@@ -1433,7 +1433,7 @@ FindSmramInfo (
         ResetWarm ();
       }
 
-      CpuDeadLoop ();
+      PANIC ("Critical HOB missing that describes MMRAM regions\n");

```

Comment:
This panic string adds a new line where the others do not. I don't think it's a big deal either way. Just wanted to call it out in case you would like it to be consistent. The `PANIC()` API states the implementation will add a newline.

Reply:
Thank you for catching the inconsistency. It has been corrected in the latest push.